docs/source/pillars/information_governance.md

```diff
@@ -156,64 +295,200 @@ _The ability of the TRE organisation to manage projects effectively._
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
+  - The unique identifier and all associated records for a user should be traceable across the entire TRE. This will include training records, affiliations, contract agreements and ethics approvals where required.
+  - Mandatory
+```
+
+## Training Delivery and Management
+
+Ability to deliver, track and maintain adequate training levels to ensure competence of all people within the TRE organisation.
+
+**Curriculum Creation and Management:** The process of designing, developing, and managing educational curricula, courses through training needs analysis for required competency.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-curriculum-creation-management
 
 * - Statement
   - Guidance
   - Importance
 * - You must have relevant training for all roles within the TRE organisation, and the ability to deliver this training.
   - This may include, for instance, cyber security training, GDPR training, and higher level training for system operators.
+    Specific training should be designed for roles such as Senior Information Risk Owner, Data Asset owner. Identification of these specialities should be done through a systematic training needs analysis.
+    Specific training may also be required based on the data or data provider such as GCP.
   - Mandatory
-* - You must ensure that users are aware which actions are and are not permitted in your TRE.
-  - This is likely to depend on the security controls in place in your TRE.
+* - You should have a training platform capable of delivering online training in a variety of formats.
+  - This should include competency testing and more simple recording of actions such as read and understood.
+  - Recommended
+* - You must provided repeat or updated where necessary to account for changes in competency requirements.
+  - Training is not a one-off event.
+    Electronic reminders for refresher training should be considered.
+    Ideally training should remain relevant and so policies and processes should enable people to demonstrate competency rather than repeat training unnecessarily.
   - Mandatory
-* - You must ensure that all users and operators of your TRE complete their training satisfactorily.
-  - This may involve assessing their level of knowledge and repeating the training if necessary.
+```
+
+**Certification Management:** The process of managing and overseeing certifications or qualifications held by individuals or entities, including tracking expiry dates, renewals, and compliance requirements.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-certification-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must maintain accurate training records that are directly tied to the role and access levels within the TRE.
+  - Electronic training records should be tied to a user record and carefully maintained.
```

Comment:
Why require electronic?

Reply:
Do you know anyone who isn't maintaining electronic records? Would you trust them with your electronic data :smile: ?

I've removed it anyway.
```suggestion
  - Training records should be tied to a user record and carefully maintained.
```